metadata_service/proxy/atlas_proxy.py

```diff
@@ -283,6 +283,11 @@ def _serialize_columns(self, *, entity: EntityUniqueAttribute) -> \
         """
         columns = list()
         for column in entity.entity[self.REL_ATTRS_KEY].get('columns') or list():
+            column_status = column.get('entityStatus', 'inactive').lower()
+
+            if column_status != 'active':
```

Comment:
will atlas do any stale column metadata pruning besides just adding the attribute? For example, we clean up the stale metadata in databuilder in https://github.com/lyft/amundsendatabuilder#removing-stale-data-in-neo4j----neo4jstalenessremovaltask .

Reply:
I think there is an endpoint to do it manually but there is no periodic flush. In majority of our searches we can just filter out inactive entities query-time, but in this it's not really an option.